source/includes/cloudstack/_isos.md

```diff
@@ -106,3 +106,48 @@ Attributes | &nbsp;
 `zoneName`<br/>*string* |The [zone id](#cloudstack-zones) that the ISO is available in. Empty if available in multiple zones
 `created`<br/>*string* | The date the template was created
 `availableInZones`<br/>*List[String]* | The list of [zone ids](#cloudstack-zones)  that the ISO is available in
+
+#### Import ISO
+
+```shell
+curl -X POST \
+   -H "Content-Type: application/json" \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/v1/services/compute-on/test_area/isos"
+```
+# Request should look like this
+```json
+{
+   "name": "windows",
+   "description":"This is my ISO",
+   "url":"http://somewhere.com/somefile.iso",
+   "zoneId":"d342520b-94aa-4e44-b066-6c18c9e4d65b"
+}
+```
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/isos</code>
+
+Import an ISO
+
+Required                   | &nbsp;
+---------------------------|-------
+`name`<br/>*string*        | The name of the ISO
+`description`<br/>*string* | The description of the ISO
+`url`<br/>*string*         | The URL where the ISO is hosted. N.B. only `http` protocol is supported
```

Comment:
Now that the web is trying to be all https, I wonder if this CloudStack limitation is a blocker

Reply:
I wouldn't say a blocker. I think if the template is protected correctly the risk is mitigated, right? It is definitely sub-optimal though